### PR TITLE
Bump SES and LavaMoat

### PR DESCRIPTION
**Pull request by FrederikBolding**
Bumps all LavaMoat packages to their latest versions.

This solves a problem where the snaps execution environment would fail to function in the latest version of Chrome Canary. It also means we can remove some of our patches as some changes have been upstreamed!

### CODE REVIEW

File: packages/snaps-execution-environments/src/common/test-utils/endowments.ts [hunk 1]
```diff
@@ -24,14 +24,14 @@ import { SILENT_LOGGER } from './logger';
  */
 export function walkAndSearch(subject: unknown, target: unknown) {
   let result = false;
-  const walker = new LavaTube(
+  walk(
```
Comment (FrederikBolding) [inline packages/snaps-execution-environments/src/common/test-utils/endowments.ts:27]: Looks like this API changed with the release of LavaTube. 

@weizman Could you take a quick look and verify this usage is still fine?

File: .yarn/patches/@lavamoat-lavapack-npm-6.0.1-1b6497ed4e.patch [hunk 2]
```diff
@@ -1,19 +1,10 @@
...
-index 5bee3d6fc752ee133aff8b12e503b674d7f32d1a..8141bff31e30d73b2f8be334272d2e20a1e73e0c 100644
+index 1ae265438785cc893e1ff707a52729c96eed9dcc..fe50b39e6c9122f299d9f81ab45c5f3e8b47926f 100644
 --- a/src/runtime.js
 +++ b/src/runtime.js
-@@ -11326,6 +11326,8 @@ function observeImports(map, importName, importIndex) {
-       errorTaming: 'unsafe',
-       // shows the full call stack
-       stackFiltering: 'verbose',
-+      // prevent issues when dealing with the "override mistake"
-+      overrideTaming: 'severe',
```
Comment (FrederikBolding) [inline .yarn/patches/@lavamoat-lavapack-npm-6.0.1-1b6497ed4e.patch:10]: This has been upstreamed to LavaMoat, simplifying our patches!

**Review by weizman — DISMISSED** [review]
Approval lavatube upgrade

**Review by Mrtenz — APPROVED** [review]

**Comment by FrederikBolding** [comment]
@SocketSecurity ignore @lavamoat/lavatube@1.0.0
@SocketSecurity ignore json-stable-stringify@1.0.2

Author is trusted!

@SocketSecurity ignore lavamoat-browserify@17.0.2
@SocketSecurity ignore @lavamoat/allow-scripts@3.0.1
@SocketSecurity ignore @lavamoat/lavapack@6.0.2
@SocketSecurity ignore lavamoat-core@15.1.1
@SocketSecurity ignore @lavamoat/aa@4.0.1
@SocketSecurity ignore lavamoat@8.0.2
@SocketSecurity ignore lavamoat-tofu@7.1.0

Packages were just published and are thus marked as "unpublished"

**Pull request merged**